pages/admin/RoleDetailsView.page.ts

```diff
@@ -0,0 +1,25 @@
+import { type Locator, Page } from '@playwright/test';
+
+export class RoleDetailsViewPage{
```

Comment:
```suggestion
export class RolleDetailsViewPage{
```
Im Client benutzen wir die deutschen Begriffe